src/web/utils/blockUtils.ts

```diff
@@ -110,4 +110,15 @@ function extendBlockStructure(
   return targetNode;
 }
 
-export {addStyleToBlock, extendBlockStructure, isBlockMarkdownType, getFirstBlockMarkdownRange};
+function getFormatType(formatCommand: string): FormatType {
+  switch (formatCommand) {
+    case 'formatBold':
+      return 'bold';
+    case 'formatItalic':
+      return 'italic';
+    default:
+      return 'underline';
```

Comment:
Not sure why underline is the default. This is not future-proof as there could be other styles as well. Maybe let's just remove this additional layer of conversion and pass `formatCommand` directly to the `formatSelection` function? This way it will be more generic.

Reply:
Updated